docs/security/trust/trust_sandbox.md

```diff
@@ -61,146 +63,114 @@ work exclusively with the Docker Hub, you would not need with these components.
-
-        $ git clone https://github.com/docker/distribution.git
-
-4. Change to the `notary` project directory.
+2. Change to the `notary` project directory.
 
         $ cd notary
 
    The directory contains a `docker-compose` file that you'll use to run a
    notary server together with a notary signer and the corresponding MySQL
    databases. The databases store the trust information for an image.
 
-5. Build the server images.
+3. Build the server images.
 
         $  docker-compose build
 
     The first time you run this, the build takes some time.
 
-6. Run the server containers on your local system.
+4. Run the server containers on your local system.
 
         $ docker-compose up -d
 
     Once the trust services are up, you'll setup a local version of the Docker
     Registry v2.
 
-7. Change to the `notarysandbox/distribution` directory.
-
-8. Build the `sandboxregistry` server.
-
-        $ docker build -t sandboxregistry .
+5. Run the `sandboxregistry` server based on the the public registry image.
 
-9. Start the `sandboxregistry` server running.
+        $ docker run -d -p 5000:5000 --net notary_sig --name sandboxregistry registry:2.4.1
 
-        $ docker run -p 5000:5000 --name sandboxregistry sandboxregistry &
+    The first time you run this, it will download the registry image from Docker Hub
+    before running it.
 
 ## Playing in the sandbox
 
-Now that everything is setup, you can go into your `notarysandbox` container and
+Now that everything is setup, you can go into your `trustsandbox` container and
 start testing Docker content trust.
 
 
-### Start the notarysandbox container
+### Start the trustsandbox container
 
-In this procedure, you start the `notarysandbox` and link it to the running
-`notary_notaryserver_1` and `sandboxregistry` containers. The links allow
+In this procedure, you start the `trustsandbox` container and link it to the running
+`notary_server_1` and `sandboxregistry` containers. The links allow
 communication among the containers.
 
 ```
-$ docker run -it -v /var/run/docker.sock:/var/run/docker.sock --link notary_notaryserver_1:notaryserver --link sandboxregistry:sandboxregistry notarysandbox
-root@0710762bb59a:/#
+$ docker run -d --privileged --net notary_sig \
+   --link sandboxregistry:sandboxregistry \
+   --link notary_server_1:notaryserver \
+   --name trustsandbox trustsandbox \
+   --insecure-registry sandboxregistry:5000
+$ docker exec -it trustsandbox sh
+/ #
 ```
 
-Mounting the `docker.sock` gives the `notarysandbox` access to the `docker`
-daemon on your host, while storing all the keys and files inside the sandbox
-container.  When you destroy the container, you destroy the "play" keys.
+Running this container, based on the docker-in-docker image, lets you pull and push images while
+storing all the trusttest images, and trust keys and files inside the sandbox container.  When
+you destroy the container, you destroy the "play" keys and the cached trusttest images.
```

Comment:
Actually, the `docker:dind` has a `VOLUME` for `/var/lib/docker`, so they are persisted in an anonymous volume. https://github.com/docker-library/docker/blob/866c3fbd87e8eeed524fdf19ba2d63288ad49cd2/1.11/dind/Dockerfile#L21